### ISSUE TITLE
Item Type Equipped trigger for offhand

### ISSUE THREAD

**Issue by Dsune0** [WeakAuras/WeakAuras2#3654]
**Is your feature request related to a problem? Please describe.**
The trigger for Item type equipped does not include offhand but every other sort of weapon

**Describe the solution you'd like**
That it would be added

**Describe alternatives you've considered**
Can make it custom ofc but seeing as every other type is included I believe offhand should be aswell

**Additional context**
![image](https://user-images.githubusercontent.com/106332935/170519637-c3b5a165-daaf-43f6-914d-a476037799d5.png)

**Comment by InfusOnWoW** [comment]
There is no special off hand weapon type. One handed swords, axes, daggers, fist weapons can be equipped in either slot.

**Comment by nullKomplex** [comment]
This is not true for the Classic era. There are plenty of items that can only be equipped in either the Main Hand or the Off Hand exclusively.

Main Hand example:
https://tbc.wowhead.com/item=34336/sunflare
Off Hand example:
https://tbc.wowhead.com/item=34949/swift-blade-of-uncertainty

Note that these are simply the first I found, the MH/OH status are not unique to simply those two weapons or even daggers.

**Comment by nullKomplex** [comment]
Alternatively the user may be referring to an actual off hand item such as:
https://tbc.wowhead.com/item=34179/heart-of-the-pit
